spec/test_prof/cops/rspec/aggregate_examples_spec.rb

```diff
@@ -0,0 +1,620 @@
+# frozen_string_literal: true
+
+require "cop_helper"
+require "test_prof/cops/rspec/aggregate_examples"
+
+RSpec.describe RuboCop::Cop::RSpec::AggregateExamples, :config do
+  subject(:cop) { described_class.new(config) }
+
+  shared_examples "detects and autocorrects in example group" do |group|
+    context "with '#{group}'" do
+      offensive_source = <<-RUBY
+        #{group} 'some docstring' do
+          it { is_expected.to be_awesome }
+          it { expect(subject).to be_amazing }
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Aggregate with the example at line 2.
+          it { expect(article).to be_brilliant }
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Aggregate with the example at line 2.
+        end
+      RUBY
+
+      good_source = <<-RUBY
```

Comment:
Now when I look at it, I'm not sure if usage of `good_source`/`offensive_source` is worth it compared to:
```ruby
expect_offense(<<~RUBY)
  ...
RUBY

expect_correction(<<~RUBY)
  ...
RUBY
```